### ISSUE TITLE
Foreign language Stripping

### ISSUE THREAD

**Issue by echodelevis** [kkoppenhaver/wp-indesign-bridge#15]
There's a way to add more caracter stripping for ''French'' language. I also noticed that none of the styles are applyed to the import. I made all caratere styling in ID both they are showed up in the import.

`content = content.replace(/<li[^>]*>/g, '').replace(/<\/li>/g, "");`  All of those stay visible after the import.

**Comment by kkoppenhaver** [comment]
@echodelevis I'll take a look at dealing with foreign languages, as this wasn't even something I had considered for v1.  Do you have any resources in particular that provide good info on this?

**Comment by echodelevis** [comment]
i will try to get you some demo about it.  ;)

**Comment by kkoppenhaver** [comment]
@echodelevis Great! With the issues that I was able to resolve today, this plugin is moving towards the first version that's actually useable, so if this is something that's breaking it, I would love to get that fixed before we start telling people about it.

**Comment by echodelevis** [comment]
Wich version of IND are you working with...I user the 2017 So cannot open Getting Started.indd

**Comment by echodelevis** [comment]
There is a way also to get the featured image from it. Possibly just the info name of it like  (images.jpg) This could be interesting function  also.

**Comment by echodelevis** [comment]
There is a way also to get the featured image from it. Possibly just the info name of it like (images.jpg) This could be interesting function also. This could be a good issues for thos making the page to know wich picture to take.

**Comment by kkoppenhaver** [comment]
Yup, that's definitely something I'm working on, as well as giving some information about images that were embedded in the post.  See https://github.com/kkoppenhaver/wp-indesign-bridge/issues/10

**Comment by echodelevis** [comment]
Great 


Envoyé depuis mon appareil Samsung

-------- Message d'origine --------
De : Keanan Koppenhaver <notifications@github.com> 
Date : 04/30/2017  3:05 AM  (GMT-03:30) 
À : kkoppenhaver/wp-indesign-bridge <wp-indesign-bridge@noreply.github.com> 
Cc : echodelevis <echodelevis@gmail.com>, Mention <mention@noreply.github.com> 
Objet : Re: [kkoppenhaver/wp-indesign-bridge] Foreign language Stripping
  (#15) 

Yup, that's definitely something I'm working on, as well as giving some information about images that were embedded in the post.  See #10

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/kkoppenhaver/wp-indesign-bridge","title":"kkoppenhaver/wp-indesign-bridge","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/kkoppenhaver/wp-indesign-bridge"}},"updates":{"snippets":[{"icon":"PERSON","message":"@kkoppenhaver in #15: Yup, that's definitely something I'm working on, as well as giving some information about images that were embedded in the post.  See https://github.com/kkoppenhaver/wp-indesign-bridge/issues/10"}],"action":{"name":"View Issue","url":"https://github.com/kkoppenhaver/wp-indesign-bridge/issues/15#issuecomment-298212652"}}}
